messages/events/debitpolicy.go

```diff
@@ -0,0 +1,33 @@
+package events
+
+// WithdrawalApprovedByDebitPolicy is an event that indicates a requested
+// withdrawal has been approved by the debit policy.
+type WithdrawalApprovedByDebitPolicy struct {
+	TransactionID string
+	AccountID     string
+	Amount        int64
+}
+
+// WithdrawalDeclinedByDebitPolicy is an event that indicates a requested
+// withdrawal has been declined due to the debit policy.
+type WithdrawalDeclinedByDebitPolicy struct {
+	TransactionID string
+	AccountID     string
+	Amount        int64
+}
+
+// TransferApprovedByDebitPolicy is an event that indicates a requested
+// transfer has been approved by the debit policy.
+type TransferApprovedByDebitPolicy struct {
+	TransactionID string
+	AccountID     string
+	Amount        int64
+}
+
+// TransferDeclinedByDebitPolicy is an event that indicates a requested
+// transfer has been declined due to the debit policy.
+type TransferDeclinedByDebitPolicy struct {
+	TransactionID string
+	AccountID     string
+	Amount        int64
+}
```

Comment:
I *think* this could be one area where we could share a `Debit[Approved|Declined]ByDebitPolicy` message across all kinds of debits, but let's keep it how you've done it for now.

Reply:
Ok, I will keep the separate ones for now and we can review it again after other changes and before merging.